### ISSUE TITLE
Possible typo on page 96?

### ISSUE THREAD

**Issue by joshuaguerin** [oreilly-qc/oreilly-qc.github.io#41]
In the explanation of Fig 5-11 on page 96, we have:
> Note that for some values of `a`, subtracting 3 from `a` will cause its _lowest_-weight qubit to be set. We can use this top bit as a condition for our increment operation.

Emphasis mine.

I'm reading this as referring to the `not`s in the `if (a<0) then b++` block being conditioned over 0x4 (for `a`). Should this be the _most_ significant qubit?

**Comment by dyordan1** [comment]
Agreed. Further, the result of the program (and the QCEngine implementation) agree with the most significant interpretation - note the 0x4 line being conditioned in the CNOTs below.
![image](https://user-images.githubusercontent.com/4634412/158734389-6c347fb1-2471-4b2c-a575-44201ff545cb.png)

This may just be a terminology clarification since the exact wording used is "lowest-weight" not "least significant".

It is also not very clear that the net result is `if(a >= -1 && a < 3) b++`. It took me quite a long time to understand that `-1-3 = -4` (duh). Perhaps `a-4` would be more clear? Or maybe even just, don't modify a for this example and setup "negative" `a` states from the beginning and skip the `a-3` / `a+3`. I felt like a bit too much was thrown into it at once and made it pretty dense to work through.
